### DISCUSSION TITLE
Add more options to history date range picker

### DISCUSSION THREAD

**Discussion by itssimon** [home-assistant/frontend#15642] [Other feature requests]
I often find myself filtering to the last hour in the History panel, and it's annoying to always have to manually pick the time range. So I was wondering if we could add a few more time range options to pick from on the left side of the dropdown.

For example:

- Last 1 hour
- Last 24 hours (different from Today, which starts at 12:00 AM)
- Last 7 days (different from This week, which begins on Sun/Mon)

![image](https://user-images.githubusercontent.com/1176585/222021321-9dd78d01-b222-4ab5-87fe-622c342ab4f8.png)

I also really like the time range picker in [Kibana](https://www.elastic.co/kibana/), where you can quickly select the last X minutes, hours or days. Maybe that could be considered for HA too?

![image](https://user-images.githubusercontent.com/1176585/222021921-8e3be660-a559-455d-90d0-7218b75e5daf.png)

**Comment by TylonHH** [comment]
Yeah... please
